unittests/platformagnostictests.py

```diff
@@ -453,6 +453,19 @@ def test_configure_removed_option(self) -> None:
             self.setconf('-Dneg_int_opt=0')
         self.assertIn('Unknown options: ":neg_int_opt"', e.exception.stdout)
 
+    def test_reconfigure_option(self) -> None:
+        testdir = self.copy_srcdir(os.path.join(self.common_test_dir, '40 options'))
+        self.init(testdir)
+        self.build()
```

Comment:
Do we actually need this to build, or is init'ing enough? (This test doesn't have any build targets.)

Reply:
build() checks that you didn't break the `build.ninja` rule (which I did :))

Reply:
Hmm not with this version of the test, so I will remove it

Reply:
For some reason I thought we validated this using ninja during backend generation but in fact we don't -- we call ninja a couple times but always ignore errors.